applications/sasquatch/values-usdfdev.yaml

```diff
@@ -123,6 +123,12 @@ kafka-connect-manager:
         connectInfluxDb: "lsst.lf"
         topicsRegex: "lsst.lf.*"
         tags: benchmark_env,module,benchmark_type
+      lsstprompt:
+        enabled: true
+        timestamp: "timestamp"
+        connectInfluxDb: "lsst.prompt"
+        topicsRegex: "lsst.prompt.*"
+        tags: dataset_tag,band,instrument,skymap,detector,physical_filter,tract,exposure,patch,visit,run
```

Comment:
IIRC tags should be values that are frequently reused. In Prompt Processing, every visit-detector is a different run, so I recommend removing `run` from the list of tags.

Prompt Processing also organizes its processing by `group`, so it may be worth including that as a tag. However, this is also an undbounded set; in general, there is a 1-1 correspondence between groups and visits (there will be slightly more groups because some visits are cancelled).

Reply:
Actually, wait -- if `run` is the name of the Butler run collection, then there is only one per night per (Gen 3) pipeline per service deployment. That might be small enough?

Reply:
That sounds good. I'll remove the `run` tag and add the new `group` tag.